src/core.js

```diff
@@ -2897,12 +2832,19 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @returns {Number}
    */
   this._getColWidthFromSettings = function(col) {
-    const cellProperties = instance.getCellMeta(0, col);
-    let width = cellProperties.width;
+    let width;
+
+    // We currently don't support cell meta objects for headers (negative values)
+    if (col >= 0) {
+      const cellProperties = instance.getCellMeta(0, col);
```

Comment:
It can be treated as a breaking change. After changing to `metaManager.getColumnMeta(col)`, 16 tests fail. Test cases something like:
```js
handsontable({
  cell: [{ row: 0, col: 2, width: '98' }, { row: 1, col: 2, width: '99' }],
});

expect(_getColWidthFromSettings(2)).toBe(98);
```
I'd suggest to not change the logic in this PR.

Reply:
Yup, it would be to easy 😉 Next time 👍